src/language-markdown/printer-markdown.js

```diff
@@ -368,6 +375,11 @@ function genericPrint(path, options, print) {
         : concat(["\\", hardline]);
     case "liquidNode":
       return replaceNewlinesWith(node.value, hardline);
+    // MDX
+    case "importExport":
+    case "jsx":
```

Comment:
Also, how do you feel about replacing these with `jsStatement` and `jsExpression`, respectively?

Reply:
These are the names from the original source (I replaced `import`/`export` with `importExport` and merged continuous `importExport` nodes so that the empty lines can be handled by the js printer), and I think these names should be easier to understand.